packages/components/src/components/CountrySelect/CountrySelect.tsx

```diff
@@ -0,0 +1,56 @@
+import type { PropsWithChildren } from "react";
+import React from "react";
+import { Select, type SelectProps } from "@/components/Select";
+import type { FlowComponentProps } from "@/lib/componentFactory/flowComponent";
+import { flowComponent } from "@/lib/componentFactory/flowComponent";
+import type { PropsWithClassName } from "@/lib/types/props";
+import { Option } from "@/components/Select/components/Option";
+import { useLocalizedStringFormatter } from "react-aria";
+import locales from "./locales/*.locale.json";
+
+export interface CountrySelectProps
+  extends PropsWithChildren<Omit<SelectProps, "children">>,
+    FlowComponentProps,
+    PropsWithClassName {
+  /** Optional flag to show DACH countries at the beginning of the list */
+  dachFirst?: boolean;
+}
+
+export const CountrySelect = flowComponent(
```

Comment:
Maybe it is sufficient to just provide a `<CountryOptions />` component, that comes with it's own properties like `sortBy` (the compare functions), or `filterBy`.